### ISSUE TITLE
Invalid HTTP headers

### ISSUE THREAD

**Issue by szepeviktor** [badges/shields#5138]
- [x] [shields.io](https://shields.io/#/)
- [ ] My own instance
- [ ] [badge-maker NPM package](https://www.npmjs.com/package/badge-maker)

**Description**

Few HTTP headers are incorrect.
https://redbot.org/?uri=https%3A%2F%2Fimg.shields.io%2Fpackagist%2Fdt%2Fszepeviktor%2Fphpstan-wordpress.svg

![kép](https://user-images.githubusercontent.com/952007/83067984-d5f68500-a067-11ea-936a-c14b20c0d841.png)

Please consider fixing them.
Thanks.

**Link to the badge**

https://img.shields.io/packagist/dt/szepeviktor/phpstan-wordpress.svg

**Comment by paulmelnikow** [comment]
Hmm, is our cache-control header missing a comma?

```
Cache-Control: max-age=120 s-maxage=120
```

Should it be?

```
Cache-Control: max-age=120, s-maxage=120
```

I'm familiar with Vary though I'm not sure what we ought to be sending for that (or if that's a problem with Cloudflare).

The `__cfduid` warning is a Cloudflare thing; not something we can easily fix.

**Comment by szepeviktor** [comment]
I think `Vary: Accept-Encoding` is the missing header.

**Comment by chris48s** [comment]
I've just deployed #5139 which fixes the cache-control header. Thanks for that @szepeviktor .
I wonder if the vary header is also cloudflare related?

https://redbot.org/?uri=https%3A%2F%2Fimg.shields.io%2Fpackagist%2Fdt%2Fszepeviktor%2Fphpstan-wordpress.svg reports the issue with Vary, but if I run the same test on staging https://redbot.org/?uri=https%3A%2F%2Fshields-staging.herokuapp.com%2F%2Fpackagist%2Fdt%2Fszepeviktor%2Fphpstan-wordpress.svg (which bypasses cloudflare - it just serves straight from heroku), we don't get that warning.

**Comment by szepeviktor** [comment]
> Server: Cowboy

Could it be that webserver is not able to compress responses??

`Accept-Encoding: gzip`

**Comment by chris48s** [comment]
Yes I think you're right :+1:  Passing `Accept-Encoding: gzip` does nothing on staging.
Are you interested in submitting a PR to add the vary header?

**Comment by szepeviktor** [comment]
>  Are you interested in submitting a PR to add the vary header?

Thank you.

I'm really deep into HTTP, but 9 on a 0-100 scale in Python.

**Comment by chris48s** [comment]
Sorry. I don't understand if that means yes or no :confused:

**Comment by szepeviktor** [comment]
a No

**Comment by chris48s** [comment]
No worries. I'll pick this one up

**Comment by PyvesB** [comment]
I don't believe we've got any header issues nowadays:
<img width="1333" height="643" alt="Image" src="https://github.com/user-attachments/assets/9972c8ae-9ade-46ad-a523-09f5afdb1eb2" />

Will go ahead and close this.
